### ISSUE TITLE
New release with C++17 support

### ISSUE THREAD

**Issue by blackccpie** [mariusmuja/flann#408]
Hi,

I noticed some PR concerning C++17 support have been merged recently, so would it be possible to have official release including theses changes?

Regards

Albert

**Comment by akashp1997** [comment]
Yes, we should look into it, as the Homebrew(OSX) also has an outdated v1.9.1 and causes errors when using C++17, which should be updated.

**Comment by elfprince13** [comment]
Bump. Please also merge #424 first.

**Comment by elfprince13** [comment]
Re-bumping.
